FakeTrello/Models/CardUserAndCreator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
```

Comment:
just like my note above, you don't need to create this class for the join table. Entity Framework will create the table based on the navigation property in your Trello User class.